content/engine/install/ubuntu.md

```diff
@@ -165,6 +165,7 @@ Docker from the repository.
    `hello-world` image.
 
    ```console
+   $ sudo service docker start
```

Comment:
The service should be running automatically after you installed the packages; what method did you use to install the packages?

Reply:
I use the command just above here in the `Latest` tab. It returns some symbolic error, but I didn't pay much attention to it. When I run the hello world command, it gives the following error: `docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the Docker daemon running?` It worked after I started the docker daemon service manually. I'm trying to install it on WSL2 Ubuntu 22.04."

Reply:
Wondering if part of the issue is WSL2; ISTR WSL2 doesn't use `systemd`, making it a non-standard setup for ubuntu (I'm not on a Windows machine though, so can't check 😅)